client/src/pages/Signup.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+
+import { Typography, Button, Paper, TextField, Grid } from "@material-ui/core";
+
+import { makeStyles } from "@material-ui/core/styles";
+import AppBarNotLoggedIn from '../layout/AppBarNotLoggedIn';
+
+const cta = {
```

Comment:
This isn't necessary to pass as a prop: it can be a part of the appbar component

Reply:
That makes sense!